libbeat/common/transport/tlscommon/ca_pinning_test.go

```diff
@@ -44,23 +44,6 @@ var ser int64 = 1
 func TestCAPinning(t *testing.T) {
 	host := "127.0.0.1"
 
-	t.Run("when the ca_sha256 field is not defined we use normal certificate validation",
```

Comment:
Why was this test removed?

Reply:
We only use normal certificate validation if verification is set to strict and `ca_sha256` field is set. I do not think this test was useful, and I choose to remove it instead of adjusting. If you believe it was useful, I am ok with keeping it.